libs/blocks/locui-create/utils/constant.js

```diff
@@ -1,6 +1,11 @@
 export const LOCALES = 'locales';
 export const LOCALE_GROUPS = 'localegroups';
 export const URL_SEPARATOR_PATTERN = /[,\r\n]/g;
+export const API_BASE_URL = {
```

Comment:
Shouldn't these constants be picked from milo config file like - https://main--bacom--adobecom.hlx.page/.milo/config.json
Reference - Loc Details Milo Page